bool setToolTip
string toolTipFrame
---
bool success
